File: 26removeDuplicatesFromSortedArray/removeDuplicatesFromSortedArray.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let index = nums.length ? 1 : 0
    nums.forEach((num, _, array) => {
        if (num > array[index - 1])
        {
            array[index] = num
            ++index
        }})
    return index
};

File: 26removeDuplicatesFromSortedArray/removeDuplicatesFromSortedArray.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int index = nums.length != 0 ? 1 : 0;
        for (final var num: nums)
            if (num > nums[index - 1])
            {
                nums[index] = num;
                ++index;
            }
        return index;
    }
}
